src/main/webapp/src/app/recommendation-form/recommendation-form.component.html

```diff
@@ -1,62 +1,65 @@
 <h1>Hidden Gems Recommendations</h1>
 <h4>Get a restaurant recommendation tailored just for you!</h4>
 
-<form [formGroup]="preferenceForm" (ngSubmit)="onSubmit()" #formID="ngForm">
+<form [formGroup]="preferenceForm" novalidate (ngSubmit)="onSubmit()" #formID="ngForm">
   <h2>Your Restaurant Preferences</h2>
   <p>Please select your restaurant preferences in the form below. If you have no preference for a 
     field, please feel free to leave it blank</p>
   
-  <h3>Price Range</h3>
-  <div  class="price-group">
+  <div  class="price-group" class="form-group">
```

Comment:
Seems like this div has 2 classes? If you need the 2 classes, I found this link: https://stackoverflow.com/questions/31412923/adding-more-than-one-class-html

But I am wondering what the purpose of the class=form-group and class=form-control is? My understanding of how forms are being controlled comes from this: https://angular.io/guide/reactive-forms. It seems these classes aren't used, but I might be missing something.

Reply:
Done - removed from file 👌